Creation of AboutMe Section

samuelouteda opened the pull request:
Assembled AboutMe component: 
![image](https://user-images.githubusercontent.com/74317636/102540077-e5c9cb00-4084-11eb-8195-43c06cb84858.png)

Assembled AboutMe component (Without Lorem Ipsum's): 
![image](https://user-images.githubusercontent.com/74317636/103236155-753f6b80-491a-11eb-9609-460920008f6f.png)

ListMaker component:
![image](https://user-images.githubusercontent.com/74317636/103236246-c4859c00-491a-11eb-8186-00b24173d24f.png)

StyledButton component:
![image](https://user-images.githubusercontent.com/74317636/102539745-781d9f00-4084-11eb-9b39-b66769080356.png)

ServicesInfo component:
![image](https://user-images.githubusercontent.com/74317636/102539832-8ff52300-4084-11eb-8bf7-27137ff1b2a1.png)

samuelouteda committed:
Creation of AboutMe Section

Creation of AboutMe Section assembling three global components and two local components: the globals are SectionTitle, SectionSubtitle, and StyledButton, a component that renders a blue button; the locals are ListMaker, which takes several values and renders a list, and ServicesInfo, which renders an icon with a title and text, all related to a specific service. An index.js was created to import all these components and be exported on ContactMe.jsx. Another changes were made, like: importing more icons on IconMaker/index.js and exporting them in IconMaker, adding logic on IconMaker to render the new icons and adding keywords in constants.js related to the icons.

samuelouteda committed:
Correcting Error

Correcting Typo Error

kilordpepo commented on src/sections/AboutMe/AboutMe.jsx:
```diff
@@ -0,0 +1,163 @@
+  ServicesInfo
+} from "./components";
+import { makeStyles, Typography, Link } from "@material-ui/core";
+import file from "../Home/components/BackgroundVid/image/image_1.jpg"; //Temporal file to test download functionality
```
can the comment be removed?

samuelouteda replied:
Of course!

kilordpepo commented on src/sections/AboutMe/components/ListMaker/ListMaker.js:
```diff
@@ -0,0 +1,72 @@
+        <li className={classes.listItem}>
+          <span className={classes.listTitle}>{text.title}</span>
+          <span className={classes.listColon}> : </span>
+          {!text.chip && text.value}
```
I would think it's bertter to put this inside a `<p>`

samuelouteda committed:
Replacing Lorem Ipsum's and improving minor details

Replacing Lorem Ipsum's in AboutMe.jsx with real information, importing EnglishCV file to be the downloadable file and putting its name in the download tag, passing keys to ServicesInfo to be used on its render method,  adding marginTop to text on ServicesInfo and using the passed keys, adding keys on ListMaker and changing numerical strings to be just numbers, correcting PropTypes on IconMaker, adding new largeThickness rules in styles.js and applying them in AboutMe.jsx

samuelouteda committed:
Removing address list element

Removing address list element from AboutMe.jsx and including key in PropTypes of ServicesInfo
